src/spec-node/devContainersSpecCLI.ts

```diff
@@ -54,6 +55,7 @@ const defaultDefaultUserEnvProbe: UserEnvProbe = 'loginInteractiveShell';
 	y.command('features', 'Features commands', (y: Argv) => {
 		y.command('test', 'Test features', featuresTestOptions, featuresTestHandler);
 		y.command('package <target>', 'Package features', featuresPackageOptions, featuresPackageHandler);
+		// y.command('publish <target>', 'Package and publish features', featuresPublishOptions, featuresPublishHandler);
```

Comment:
Commented until we have wired in changes from https://github.com/devcontainers/cli/pull/99

Reply:
I got my changes in faster than I expected, so no need to comment this out anymore 😎